cost/azure/old_snapshots/azure_delete_old_snapshots.pt

```diff
@@ -0,0 +1,225 @@
+name "Azure Old Snapshots"
+rs_pt_ver 20180301
+type "policy"
+short_description "Checks for snapshots older than specified number of days and, optionally, deletes them. See the [README](https://github.com/flexera/policy_templates/tree/master/cost/azure/old_snapshots) and [docs.rightscale.com/policies](https://docs.rightscale.com/policies/) to learn more."
+severity "low"
+category "Cost"
+info(
+  version: "2.0",
+  provider: "Azure",
+  service: "Storage",
+  policy_set: "Old Snapshots"
+  )
+
+##################
+# User inputs    #
+##################
+
+parameter "param_email" do
+  type "list"
+  label "Email addresses of the recipients you wish to notify"
+end
+
+parameter "param_snapshot_age" do
+  type "number"
+  label "The number of days since the snapshot was created"
+end
+
+parameter "param_exclusion_tag_key" do
+  category "User Inputs"
+  label "Exclusion Tag Key"
+  description "Cloud native tag key to ignore instances. Example: exclude_utilization"
+  type "string"
```

Comment:
this should be a list.  and make sure you update the code to evaluate the list not a string

Reply:
Made it as list type.